### PR TITLE
fixes #566 : custom-menu-content not showing up 

### LINKED ISSUE #566: guide.hbs 's custom-menu-content not showing up when clicked.
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

**Expected behavior:**

custom-content-menu `hidden` property is removed on clicking, similar to its behaviour on home page i.e. badgeyay.com 
**Steps to reproduce:**
go to badgeyay.com/#/guide, custom-content-menu not coming when clicked.
image

**I am not working on it**.

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes #566 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

## preview
https://pr-567-fossasia-badgeyay.surge.sh/

#### Changes proposed in this pull request:
-Now custom-menu-content shows up when clicked on

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@nityanandagohain Issue is still the same :(

File: frontend/app/templates/guide.hbs [hunk 1]
```diff
@@ -25,4 +25,10 @@
...
     </div>
-</div>
\ No newline at end of file
+</div>
+
+
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
+<script src="{{rootURL}}js/jquery.minicolors.min.js"></script>
+<script src="{{rootURL}}js/script.js"></script>
+<script src="{{rootURL}}js/form.js"></script>
```
Comment (abishekvashok) [inline frontend/app/templates/guide.hbs]: We don't need this here.

**Comment by nityanandagohain** [comment]
done @vaibhavsingh97

File: frontend/app/templates/guide.hbs [hunk 2]
```diff
@@ -25,4 +25,10 @@
             <img src="/images/guide_manual.png" class="guide_img" alt="Manual Data Image">
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
+<script src="{{rootURL}}js/jquery.minicolors.min.js"></script>
```
Comment (abishekvashok) [inline frontend/app/templates/guide.hbs]: We don't need this one here

File: frontend/app/templates/guide.hbs (hunk 1 above)
Comment (nityanandagohain) [reply frontend/app/templates/guide.hbs]: I have removed form.js and jquery but script.js is required otherwise it is not working

**Comment by rupav** [comment]
@sagar-kalra already an issue is opened for it https://github.com/fossasia/badgeyay/issues/565 . Please see

**Comment by sagar-kalra** [comment]
@rupav sorry for that
didnt see that issue

**Comment by sagar-kalra** [comment]
okay @rupav 
sorry for pointing it out on here

**Comment by rupav** [comment]
no problem @sagar-kalra

**Comment by abishekvashok** [comment]
Thanks!

**Pull request merged**